### PR TITLE
Internet of things tests - Content

### PR DESCRIPTION
**Pull request by ysukhova**
Content tests for https://staging.tech-stack.com/services/internet-of-things
image

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+		'4.Data management layer',
+		'5.Presentation layer',
+		'6.Testing & processes'
+	];
+
+	expect(actualSectionTitles).toEqual(expectedSectionTitles);
+});
+
+test('Check section titles and sencor image in "IoT Engineering Case Studies" container. @Regression @IoTEngineeringServices @TSWEB-695', async () => {
+	const ioTEngineeringCaseStudiesContainer = driver.getByTestId(IoTEngineeringServices.ioTEngineeringCaseStudies);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts:150]: mb just
Check section titles and image....
OR
sen**s**or

File: ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+		'Real-time monitoring of production processes',
+		'Valuable insights on process adjustments',
+		'Cost savings from addressing over/undercooling',
+		'Forecasting capabilities',
+		'Process improvements'
+	];
+
+	expect(actualSectionTitles).toEqual(expectedSectionTitles);
+	expect(driver.getByTestId(IoTEngineeringServices.sensorsScheme).isVisible());
+});
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts:162]: could you use
await expect(driver.getByTestId(IoTEngineeringServices.sensorsScheme)).toBeVisible();

File: ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+	expect(actualSectionTitles).toEqual(expectedSectionTitles);
+});
+
+test('Check section titles and sencor image in "IoT Engineering Case Studies" container. @Regression @IoTEngineeringServices @TSWEB-695', async () => {
+	const ioTEngineeringCaseStudiesContainer = driver.getByTestId(IoTEngineeringServices.ioTEngineeringCaseStudies);
+	const actualSectionTitles = await ioTEngineeringCaseStudiesContainer.getByTestId(Container.SectionTitle).allInnerTexts();
+	const expectedSectionTitles = [
+		'Real-time monitoring of production processes',
+		'Valuable insights on process adjustments',
+		'Cost savings from addressing over/undercooling',
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts:155]: pls check also numbers and IoT "humidity sensors scheme" block title

File: ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+	const expectedSectionTitles = [
+		'Investigation',
+		'Execution',
+		'Performance',
+		'Analysis'
+	];
+
+	expect(actualSectionTitles).toEqual(expectedSectionTitles);
+	expect(await ioTEngineeringProcessContainer.getByTestId(Container.BlockTitle).textContent()).toEqual('New IoT Device\nEngineering');
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/services/ioTEngeneeringServices/IoTEngeneeringServicesContent.test.ts:197]: pls check carousel numbers and SectionTitles in "New IoT Device Engineering" block

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

**Review by anastasiia-liashova — APPROVED** [review]

**Pull request merged**